packages/design-system/src/types/index.d.ts

```diff
@@ -28,6 +27,7 @@ export * from './FilterChip';
 export * from './FormControl';
 export * from './FormLabel';
 export * from './InlineError';
+export * from './MonthPicker';
```

Comment:
Does the `MonthPicker.d.ts` file need to be deleted?

Reply:
Good catch!